ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -6368,3 +6368,110 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
```

Comment:
@varungur the jobs in ci-operator/jobs/openshift/relase are for the openshift/release repo. Please put the job in ci-operator/jobs/$your-org/$your-repo

Reply:
The goal is that this becomes a release informing job, is this not the correct place for that?

Reply:
Okay, seems they all got cargoculted into this dir even though they don't belong there. We will do a follow-up to give all of them their distinct location.

it is mostly wrong because the ppl in the owners file for that dir are not the right ones to review this.

/approve
/uncc